src/main/webapp/app/faq/faq.component.ts

```diff
@@ -25,32 +28,40 @@ import { SearchFilterComponent } from 'app/shared/search-filter/search-filter.co
     imports: [ArtemisSharedModule, CustomExerciseCategoryBadgeComponent, ArtemisSharedComponentModule, ArtemisMarkdownModule, SearchFilterComponent],
 })
 export class FaqComponent implements OnInit, OnDestroy {
+    protected readonly FaqState = FaqState;
     faqs: Faq[];
+    course: Course;
     filteredFaqs: Faq[];
     existingCategories: FaqCategory[];
     courseId: number;
     hasCategories: boolean = false;
+    isAtLeastInstructor: boolean = false;
```

Comment:
I haven't noticed on the first review, in the future please always introduce signals. For this little change I do not mind